web/locales/fr.json

```diff
@@ -1232,17 +1232,19 @@
         },
         "RU-1": {
             "countryName": "Russie",
-            "zoneName": "Russie européenne et Ural"
+            "zoneName": "Europe-Oural"
         },
         "RU-2": {
             "countryName": "Russie",
             "zoneName": "Sibérie"
         },
         "RU-EU": {
-            "zoneName": "Russie"
+            "countryName": "Russie",
+            "zoneName": "Arctique"
         },
```

Comment:
Would you mind also doing this for the English language file? It's nice to have actual zones and not just the country name for all of them :)

Reply:
No problem